qdrant-landing/content/blog/qdrant-1.13.x.md

```diff
@@ -0,0 +1,430 @@
+
+### Our New Storage Architecture
+
+Storage is divided into three layers. The **Data Layer**, **Mask Layer** and **Tracker Layer**.
+
+{{< figure src="/blog/qdrant-1.13.x/storage.png" alt="Qdrant's New Storage Backend" caption="Qdrant's New Storage Backend" >}}
+
+**The Data Layer** consists of fixed-size blocks that store the actual data. The block size is a configurable parameter that can be adjusted based on the workload. Each record occupies the required number of blocks. If the data size exceeds the block size, it is split into multiple blocks. If the data size is smaller than the block size, it still occupies an entire block.
+
+**The Mask Layer** contains a bitmask that indicates which blocks are occupied and which are free. The size of the mask corresponds to the number of blocks in the Data Layer. For instance, if the block size is 128 bytes, the bitmask will allocate 1 bit for every 128 bytes in the Data Layer. This results in an overhead of 1/1024 of the Data Layer size. The bitmask is stored on disk and does not need to be loaded into memory.
+Furthermore, there is an additional structure which tracks gaps in regions of the bitmask. This is to get an even smaller overhead against the data, which can be loaded into memory easily. Each region summarizes 1KB of bits in the bitmask, which represents a millionth scale of the Data Layer size, or 6 KB of RAM per GB of data.
+
+**The Tracker Layer** is in charge of fast lookups, it directly links the IDs of the points to the place where the data is located.
+
+## GPU Accelerated Indexing 
+
+![gpu-accelerated-indexing](/blog/qdrant-1.13.x/image_6.png)
+
+We are making it easier for you to handle even **the most demanding workloads**.
+
+Qdrant now supports GPU-accelerated HNSW indexing **on all architectures, including NVIDIA, AMD, Intel**. This new feature dramatically reduces indexing times, making it a game-changer for projects where speed truly matters.
+
+> Indexing over GPU now delivers speeds up to 10x faster than CPU-based methods for the equivalent hardware price.
+
+Our custom implementation of GPU-accelerated HNSW indexing **is built entirely in-house**. Unlike solutions that depend on third-party libraries, our approach is vendor-agnostic, meaning it works seamlessly with any modern GPU that supports **Vulkan API**. This ensures broad compatibility and flexibility for a wide range of systems.
+
+*Here is a picture of us, running Qdrant with GPU support on a SteamDeck (AMD Van Gogh GPU):*
+
+{{< figure src="/blog/qdrant-1.13.x/gpu-test.jpg" alt="Qdrant on SteamDeck" caption="Qdrant on SteamDeck with AMD GPU" >}}
+
+This experiment didn't require any changes to the codebase, and everything worked with the default Docker image.
```

Comment:
GPU support requires a different Docker image, not `latest` but `gpu-nvidia-latest` or `gpu-amd-latest`. I don't suggest to explicitly mention those here, but that means it's not the 'default' Docker image.

```suggestion
This experiment didn't require any changes to the codebase, and everything worked right out of the box with our AMD Docker image.
```